power_grid_model_c/power_grid_model/include/power_grid_model/common/statistics.hpp

```diff
@@ -207,6 +207,24 @@ template <symmetry_tag sym_type> struct PolarComplexRandVar {
     }
 };
 
+template <symmetry_tag sym, template <symmetry_tag> typename RandVarType>
+    requires std::same_as<RandVarType<sym>, UniformComplexRandVar<sym>> ||
+             std::same_as<RandVarType<sym>, IndependentComplexRandVar<sym>>
+inline auto conj(RandVarType<sym> var) {
+    var.value = conj(var.value);
+    return var;
+}
+
+template <symmetry_tag sym> inline auto conj(DecomposedComplexRandVar<sym> var) {
+    var.imag_component.value = -var.imag_component.value;
+    return var;
+}
+
+template <symmetry_tag sym> inline auto conj(PolarComplexRandVar<sym> var) {
+    var.angle.value = -var.angle.value;
+    return var;
+}
+
```

Comment:
So you receive argument by value (as a copy) and then modify the `conj` in-place?